.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines in the [CONTRIBUTING.md](https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md) file and learn how to compile Falco from source [here](https://falco.org/docs/source).
+2. Please label this pull request according to what type of issue you are addressing.
+3. Please add a release note!
+4. If the PR is unfinished while opening it specify a wip in the title before the actual title, for example, "wip: my awesome feature"
+-->
+
+**What type of PR is this?**
+
+> Uncomment one (or more) `/kind <>` lines:
+
+> /kind bug
```

Comment:
**todo:** also update [test-infra's config](https://github.com/falcosecurity/test-infra/blob/d8925b88d0ba19d40ad5167a5c1397a95019b8aa/config/plugins.yaml#L130) for labels in this repo.

Reply:
Done! ✔️ [https://github.com/falcosecurity/test-infra/pull/971](https://github.com/falcosecurity/test-infra/pull/971)